Gemfile

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
+
+# gem "rails"
```

Comment:
If you're not using any gems, you don't need to run bundle. Gemfile and Gemfile.lock unnecessary bloat.

Reply:
True, thank you